data/json/items/tool/fire.json

```diff
@@ -137,6 +137,23 @@
     "use_action": { "type": "firestarter", "moves": 40, "moves_slow": 1000 },
     "flags": [ "FIRESTARTER", "NO_RELOAD", "NO_UNLOAD", "WATER_BREAK" ]
   },
+  {
+    "id": "ref_matches",
+    "type": "TOOL",
+    "name": { "str": "matchbox" },
+    "description": "This is a small box of matches with a coarse strike surface on the side.  Matches must be carried to use various drugs like cigarettes, or to light things like Molotov cocktails.  You can also use matches to light nearby items on fire.",
+    "weight": "10 g",
+    "volume": "10 ml",
```

Comment:
Based on some quick googling, I would bump the volume up to 30 ml for the matchbox, but feel free to look around for a specific example to base this on. This might also mean it could hold a few more matches, but again, I'll let you decide if that makes sense for the example you choose to base this on.

Reply:
After i look around, i found matchbox with volume 24.6ml can contain 32 match but i haven't found the 30ml yet, probably the 30ml matchbox can contain around 50 match.

Reply:
> 24.6ml can contain 32 match

Those seem like reasonable numbers and still gives it the advantages of holding more and being reloadable, while being slightly bigger. The one I found was probably just a guesstimate of size or average of multiple variants.

Reply:
i will make proper changes, thanks for mentioning the volume.